src/main/java/org/zalando/nakadi/controller/NakadiController.java

```diff
@@ -0,0 +1,74 @@
+package org.zalando.nakadi.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import org.zalando.nakadi.domain.ItemsWrapper;
+import org.zalando.nakadi.security.Client;
+import org.zalando.nakadi.service.FloodService;
+import org.zalando.nakadi.util.FeatureToggleService;
+
+import static org.zalando.nakadi.security.Client.FULL_ACCESS;
+
+@RestController
+@RequestMapping(value = "/nakadi")
```

Comment:
why `/nakadi` ?
We already have "nakadi" in our URL in the very beginning. I think it should be more meaningful. `/nakadi` means everything and nothing at the same time.